### PR TITLE
Add Windows service support

### LINKED ISSUE #83: Run Duck as a Windows Service
**Is your feature request related to a problem? Please describe.**
Would be nice to be able to run Duck as a Windows Service

### PR DESCRIPTION
**Pull request by patriksvensson**
* Do not block on HTTP server
* Add support for installing and uninstalling service

Closes #83

### CODE REVIEW

File: build.rs [hunk 1]
```diff
@@ -6,7 +6,7 @@ fn main() {
     if env::var("CARGO_FEATURE_EMBEDDED_WEB").is_ok() {
         let dir = Path::new("./web/dist");
         if !dir.exists() {
-            panic!("The UI have not been built.");
+            panic!("The UI have not been built");
```
Comment (gep13) [inline build.rs:9]:
```suggestion
            panic!("The UI has not been built");
```

**Comment by gep13** [comment]
@patriksvensson ah, see that you already merged this :-)  Nothing major in the review.

**Comment by patriksvensson** [comment]
@gep13 Not to late to change things. What was your suggestion?

**Comment by patriksvensson** [comment]
@gep13 Didn’t see your comment for some reason. Will change! Thanks a lot! 😊

**Pull request merged**